### ISSUE TITLE
Its too hard to get rid of dead rat king servants

### ISSUE THREAD

**Issue by lzk228** [space-wizards/space-station-14#29432]
title

**Comment by Brandon-Huu** [comment]
can't put them in your backpack?

**Comment by XxSWAG-MASTER-OVERLORDxX** [comment]
> can't put them in your backpack?

No, they are rats not mice. They can only be butchered or dragged into a dispo bin

**Comment by SabatonShip** [comment]
How about making it so they can be picked up then, like mices? And even eaten? The reptilians will be pleased.

**Comment by UbaserB** [comment]
> How about making it so they can be picked up then, like mices? And even eaten? The reptilians will be pleased.

that would pose issues of being able to also pick them up alive, but i also agree on this

**Comment by SabatonShip** [comment]
> > How about making it so they can be picked up then, like mices? And even eaten? The reptilians will be pleased.
> 
> that would pose issues of being able to also pick them up alive, but i also agree on this

Yes! But still why would you do that, it will bite you to death xD
